components/screens/DeployToken.tsx

```diff
@@ -0,0 +1,406 @@
+import Header from "@/components/GeneralHeader";
```

Comment:
It would be better to put this page to `full-page` folder.

Reply:
dbcde9e17b2b8c8a5e7bd08be38d5d506434fa80